states/MA/governor.yaml

```diff
@@ -0,0 +1,51 @@
+# Note that sometimes a feedback box pops up upon initial load of the page.  This doesn't show up every time, and doesn't appear twice in a row, so resubmissions will be successful if the feedback box is first encountered.
+
+contact_form:
+  steps:
+    - visit: "https://www.mass.gov/forms/email-the-governors-office"
+    - fill_in:
+        - name: "field49751862"
+          selector: "#field49751862"
+          value: $NAME_FIRST
+          required: true
+        - name: "field49751877"
+          selector: "#field49751877"
+          value: $NAME_LAST
+          required: true
+        - name: "field49751885"
+          selector: "#field49751885"
+          value: $ADDRESS_STREET
+          required: true
+        - name: "field49751935"
+          selector: "#field49751935"
+          value: $ADDRESS_CITY
+          required: true
+        - name: "field49751937"
+          selector: "#field49751937"
+          value: $ADDRESS_STATE_FULL
```

Comment:
I'd probably use the two-letter abbrev but obvs either is fine.

Reply:
I'll change it.